sympy/calculus/singularities.py

```diff
@@ -136,29 +171,40 @@ def is_strictly_decreasing(f, interval=S.Reals):
     Examples
     ========
 
-    >>> from sympy import is_decreasing
-    >>> from sympy.abc import x
+    >>> from sympy import is_strictly_decreasing
+    >>> from sympy.abc import x, y
     >>> from sympy import S, Interval, oo
-    >>> is_decreasing(1/(x**2 - 3*x), Interval.open(1.5, 3))
+    >>> is_strictly_decreasing(1/(x**2 - 3*x), Interval.open(1.5, 3))
```

Comment:
i have changed `is_decreasing` -> `is_strictly_decreasing`  here and in tests as well. These were probably left by mistake. @aktech can you verify. I think created these functions.